master/internal/config/authz_config.go

```diff
@@ -78,8 +78,9 @@ func (a AssignWorkspaceCreatorConfig) Validate() []error {
 
 // IsRBACUIEnabled returns if the feature flag RBAC should be enabled.
 func (c AuthZConfig) IsRBACUIEnabled() bool {
+	_, isRBACCapable := knownAuthZTypes["rbac"]
 	if c.RBACUIEnabled != nil {
-		return *c.RBACUIEnabled
+		return *c.RBACUIEnabled && isRBACCapable
```

Comment:
This does prevent you from using user groups feature in OSS now right? I doubt anyone was using it just want to double check this is intentional since the PR calls out test only changes.

Reply:
You're totally right. A more principled approach would have committed this in two separate PRs. I'll make sure the squashed commit message spells this out.